### PR TITLE
Shrink acceptance test data - PR9

### PR DESCRIPTION
**Pull request by bayliffe**
A component of https://github.com/metoppv/improver/issues/1804

To reduce the data volume of the acceptance test data repository, work has been done to coarsen the resolution of test data relating to the following CLIs:

- spot-extract
- neighbour-finding

This PR updates the checksums and removes a redundant test for spot-extract. It also corrects a doc-string.
The changed acceptance test data is fundamentally changed due to the nature of these tests and needs careful review.

The relevant acceptance test data can be found in branch shrink_atd_9
Plotting of the original and modified data [can be seen here](https://github.com/MetOffice/improver_aux/blob/acceptance_test_data_shrinking/notebooks/acceptance_test_data/shrink_atd_9.ipynb).

### CODE REVIEW

**Review by brhooper — CHANGES_REQUESTED** [review]
For the most part, this seems fine, however the following 4 files seem to contain the same data now:

neighbour-finding/outputs/nearest_global_kgo.nc
neighbour-finding/outputs/nearest_land_constraint_minimum_dz_global_kgo.nc
neighbour-finding/outputs/nearest_land_global_kgo.nc
neighbour-finding/outputs/nearest_minimum_dz_global_kgo.nc

which was not the case previously. I don't think that this is expected behaviour as each kgo should be testing different functionality. The sites don't seem to have been changed much, if at all, so I assume that this is caused by the coarser resolution of the inputs.

**Comment by bayliffe** [comment]
@brhooper you should now find that the data are different in all of the KGO. I have significantly increased the search radius for the nearest land or similar altitude neighbour to achieve this for the global data on the coarse grid. The figure below, covering some of South America, shows sites (circles) and the chosen neighbouring grid points (triangles) that correspond to different options (e.g. nearest, nearest-land, etc); sites and associated neighbours share the same colours.


spot_options

**Review by brhooper — APPROVED** [review]
Thanks for making those changes @bayliffe, this all looks fine now.

**Comment by MoseleyS** [comment]
@bayliffe: Please can you resolve the conflicts on this PR?

File: improver_tests/acceptance/SHA256SUMS [hunk 1]
```diff
@@ -553,33 +552,31 @@ efff8720d452591b6531a279bad6c0168db2336fa497c1f1087ecfcb9321af92  ./sleet_probab
...
+c14e6b835baa4427ad129a5dc3bfbf47b28400527a90013e1240c2f158313bdc  ./spot-extract/outputs/extract_resampled_percentiles.nc
+6339135535fd59756df6c91665b4cfdc92d8dd1527e600363d6067feac682b0c  ./spot-extract/outputs/lapse_adjusted_multiple_percentile_kgo.nc
+ec190e3249be6588f54d44061038257e86a9f3f9fee2e1690ddcc1315f9dd379  ./spot-extract/outputs/lapse_rate_adjusted_uk_temperatures.nc
+8acde1c5b6e152e71fcfe981dae3e425d8b59c891997c998f9d6cb2cf67a6ac4  ./spot-extract/outputs/mindz_land_constraint_uk_temperatures.nc
+d24b59b25252796d6ee2f26e7e6764b84145957b0d32a4246e075db931b7b46c  ./spot-extract/outputs/mindz_uk_temperatures.nc
+332165b98af3eac6cd56c30457cc9c031b9788a1f660295a06b0108b40816b57  ./spot-extract/outputs/multi_time_kgo.nc
+4cf5c380cff65016b32bd852e31a9ed3ea4ff60fcf879e75f760c63fac27c58d  ./spot-extract/outputs/nearest_uk_temperatures.nc
+5d68bab5444b1ee2c1ad6179bf19ba58cfc1cea60bc9684b0100c5c0035d4b74  ./spot-extract/outputs/nearest_uk_temperatures_unique_ids.nc
+23007b89be636fb0e7445f239a30fc9debf530fc19e33599a09226ec31625e7e  ./spot-extract/outputs/with_realization_collapse.nc
```
Comment (MoseleyS) [inline improver_tests/acceptance/SHA256SUMS:579]: This output file contains only zeroes. So did the previous version. Basically, it isn't raining in the source data. Can you replace the source data so this task does something meaningful?

**Review by MoseleyS — CHANGES_REQUESTED** [review]
I've spotted one test that doesn't have any non-zero data in it, so I'm requesting that it is changed. Otherwise, these changes look good.

File: improver_tests/acceptance/SHA256SUMS [hunk 2]
```diff
@@ -553,33 +552,31 @@ efff8720d452591b6531a279bad6c0168db2336fa497c1f1087ecfcb9321af92  ./sleet_probab
...
-eff5c671c33c3520e9e7f0bcab4fe2a4eb5557fd84767da1a8c494f3ab7f3aa6  ./spot-extract/outputs/mindz_uk_temperatures.nc
-99af8bc4a2d6dab2da309326aed3d762f72d945fe9e864c5056369f2747cd85b  ./spot-extract/outputs/multi_time_kgo.nc
-522f5331735a778f65ad5ed6ca5ebca03bdcb8d4c7a783d6f7afe61b61efa0ab  ./spot-extract/outputs/nearest_uk_temperatures.nc
-522f5331735a778f65ad5ed6ca5ebca03bdcb8d4c7a783d6f7afe61b61efa0ab  ./spot-extract/outputs/nearest_uk_temperatures_amended_metadata.nc
-7f7734eb79abe416b0587cff2fbff286854e978037be3d63054c93d67b8bdc39  ./spot-extract/outputs/nearest_uk_temperatures_unique_ids.nc
-bf26c1be428a207a20d3a2b54987b24435944aaf52135e297176cc11f3fc3b97  ./spot-extract/outputs/with_realization_collapse.nc
+d4dcdeb1ed121db2c9e96db11283d20a80ab5924ee82b750cab8b02342e16249  ./spot-extract/inputs/all_methods_uk.nc
+5a5e97eeaf2fd457f0e2b655e1b5a0e4befc0176fa1d45d04e7c7439ea9ccedb  ./spot-extract/inputs/all_methods_uk_unique_ids.nc
+1f83848537fb5b7fa11708ed3b12966d8c6850fc97f2616411646e7a61ea4a7d  ./spot-extract/inputs/enukx_lapse_rate.nc
+a71031207447765f56f1c6a1d03342bfbd4363f4f170d191eb690c54d3fe8082  ./spot-extract/inputs/enukx_preciprate_realizations_thresholds.nc
```
[the remark this answers is not in the document]
Comment (bayliffe) [reply improver_tests/acceptance/SHA256SUMS:558]: Renamed `precipacc` after recreating it with more accumulation data.

**Review by MoseleyS — APPROVED** [review]
Looks better now. Assigning to Ben to resolve the conflicts before merging.

**Review by MoseleyS — APPROVED** [review]

**Pull request merged**